### DISCUSSION TITLE
Is it possible to recieve a sample of the data

### DISCUSSION THREAD

**Discussion by IlanMeiss** [akdasUAF/ForestFireDetection#17] [Q&A] [answered]
Hey! I would like to know if it's possible to get a subset of the data we will be working on so I can be able to get familiar with it and to run and get to know the model myself. I'm also interested in knowing how large is the dataset you are working with.

Thanks,
Ilan

**Comment by IlanMeiss** [comment]
Also, what do you sugest should be the first steps I should take to get aquainted with the project before making my project proposal.

Thanks.

**Reply by YaliWang2019** [reply, marked as answer]
Hi Ilan,

The dataset we are using is from the IEEE FLAME Dataset:
https://ieee-dataport.org/open-access/flame-dataset-aerial-imagery-pile-burn-detection-using-drones-uavs
They have different image folders for training and validation. We have tried both RGB and thermal images from this dataset. For this project, it is highly recommended to use their RGB images.

Thanks,
Yali Wang
